vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1257,6 +1258,89 @@ public Call modifyVideoInAlbums(@NotNull final Video video,
 
     // </editor-fold>
 
+    // -----------------------------------------------------------------------------------------------------
+    // Connected Apps
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Connected Apps">
+
+    /**
+     * Gets a {@link com.vimeo.networking2.ConnectedAppList} for the authorized user.
+     *
+     * @param cacheControl The cache control.
+     * @return A {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedAppList}.
+     */
+    @NotNull
+    public Call<com.vimeo.networking2.ConnectedAppList> getConnectedApps(@NotNull final CacheControl cacheControl) {
+        return mVimeoService.getConnectedApps(getAuthHeader(), createCacheControlString(cacheControl));
+    }
+
+
+    /**
+     * Returns a {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedApp}.
+     *
+     * @param type         The {@link ConnectedAppType} of the {@link com.vimeo.networking2.ConnectedApp} to return.
+     * @param cacheControl The cache control.
+     * @return A {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedApp}.
+     */
+    @NotNull
+    public Call<com.vimeo.networking2.ConnectedApp> getConnectedApp(@NotNull final ConnectedAppType type,
+                                                                    @NotNull final CacheControl cacheControl) {
+        return mVimeoService.getConnectedApp(getAuthHeader(),
+                                             type.getValue(),
+                                             createCacheControlString(cacheControl));
+    }
+
+    /**
+     * Creates a {@link com.vimeo.networking2.ConnectedApp} for the authorized user for a specific
+     * {@link ConnectedAppType}.
+     *
+     * @param type          The {@link ConnectedAppType} of the {@link com.vimeo.networking2.ConnectedApp} to create.
+     * @param authorization An authorization string for the third party platform. The nature of the string will
+     *                      vary depending upon the {@link ConnectedAppType} but should never be
+     *                      empty or the request will fail.
+     * @return A {@link Call} object or null if a client-side initialization error has occurred due to an invalid type
+     * or authorization argument.
+     */
+    @Nullable
+    public Call<com.vimeo.networking2.ConnectedApp> createConnectedApp(@NotNull final ConnectedAppType type,
```

Comment:
Minor: Add an import for the `ConnectedApp` so the class doesn't have to use the fully qualified name here or in the docs above.